File: ympa_fsharp_functions_server/src/model/GetCampaignSettingsResponse.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.CampaignSettingsDTO

module GetCampaignSettingsResponse =

  //#region GetCampaignSettingsResponse

  [<CLIMutable>]
  type GetCampaignSettingsResponse = {
    [<JsonProperty(PropertyName = "settings")>]
    Settings : CampaignSettingsDTO;
  }

  //#endregion

File: src/gatling/scala/org/openapitools/client/model/GetCampaignSettingsResponse.scala

package org.openapitools.client.model


case class GetCampaignSettingsResponse (
    _settings: Option[CampaignSettingsDTO]
)
object GetCampaignSettingsResponse {
    def toStringBody(var_settings: Object) =
        s"""
        | {
        | "settings":$var_settings
        | }
        """.stripMargin
}
